packages/api-reference/src/components/GettingStarted.vue

```diff
@@ -28,6 +28,20 @@ const themeIds: ThemeId[] = [
   'deepSpace',
 ]
 
+const themeNames: Record<ThemeId, string> = {
```

Comment:
I would be sweet to actually put these in `@scalar/themes` so we could use them everywhere 😍

Reply:
will do!